tune_sklearn/tune_basesearch.py

```diff
@@ -15,7 +15,7 @@
 from sklearn.base import BaseEstimator
 from sklearn.utils.validation import check_is_fitted
 from sklearn.model_selection import check_cv
-from sklearn.metrics import check_scoring
+from sklearn.metrics._scorer import _check_multimetric_scoring
```

Comment:
same with this - this is a private method and we should be careful depending on it. Let's move it out?